src/sbl_filing_api/main.py

```diff
@@ -42,6 +45,11 @@ def run_migrations():
 app = FastAPI(lifespan=lifespan)
 
 
+@repeat_every(seconds=settings.expired_submission_check_secs, wait_first=True, logger=log)
+async def check_expired_submissions() -> None:
+    await submission_repo.check_expired_submissions()
```

Comment:
I think for MVP, this is fine; but in the grand scheme of things, this should probably be part of another non-api service that deals with collector type of tasks.
food for thought, another approach we might be able to do is with task timeout wrapper, something like
```python
async with asyncio.timeout_at(asyncio.get_running_loop().time() + timeout_in_seconds):
    ...
```

Reply:
Yeah this is definitely purely an MVP implementation to handle the 'just in case' situations where an exception or scenario we didn't think about creeps up and causes the frontend to continually spin because the latest submission never gets out of the in progress state.   

I went with the single task polling the db route instead of a thread for each submission that expires that particular one ( assuming that is what you're alluding to there).  I've done both in the past and had less headaches with a single periodic updater.  I'm good to discuss more before squashing and merging.  But yes, this is a temporary solution to an issue that more robust error handling recently implemented will probably take care of but gives the frontend warm fuzzies ;)